hr_family/README.rst

```diff
@@ -0,0 +1,76 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================
+Employee Family Information
+===========================
+
+This module allows you to enter extra information about employee's family.
```

Comment:
Can you elaborate on the use cases of this module? I don't see a reason why somebody should use this module.

Reply:
I may jump in here: some employees get presents for their children's birthdays. Yet I'm not sure if this module should be as-is, since there may be different types of relations and probably it would be better to refactor this module to have `family_member` model with `relation_type`, and rest of fields.

@pedrobaeza what would be a procedure if there's a different way for getting similar features?

Reply:
Well, the idea is to get the best solution for a problem. Sometimes the answer is not trivial, and depends on use cases, implementation budget, etc.

For this case, indeed it's a very specific case that was implemented quickly, but it's very uncommon. If you ask me, I would go for a generalization using the same scheme as `partner_multi_relation`, defining relations and linking to partners, but this time having one of the sides of the relation being the employee, and then you can make use of all the possible interesting information in the partner.

Reply:
Maybe to proceed in-between, since employee family being in contacts/partners seems overhead? Usually it's name + phone + DOB + sex w/o extra information?

Reply:
the effort to make a more generic hr_family is in here https://github.com/OCA/hr/pull/489 thanks @alexey-pelykh, we can continue this conversation over there

Reply:
I'd keep migration separate from refactor

Reply:
I agree with you, this should keep as a migration, and #489 should be an improvement, I deleted everything from this change that is not part of the migration